### ISSUE TITLE
In the HTML debug online testing mode, custom data for request parameters is invalid.

### ISSUE THREAD

**Issue by q1763868326** [TongchengOpenSource/smart-doc#74]
```
    /**
     * Get the status of each point for a specified user under the project
     * @param projectId Project ID
     * @param openId Mini Program User OpenId|ok40T1Q0XtkL6nZLDkZ2fgCCX58c
     * @return
     */
```
The documentation shows that the parameter values are written like this.

However, the parameters in Query-parameters: are invalid and only work in Curl-example:
![image](https://user-images.githubusercontent.com/42771347/103071278-406fa380-45fe-11eb-8d6d-e12a7b8b81e2.png)
![image](https://user-images.githubusercontent.com/42771347/103071324-53827380-45fe-11eb-9d81-3b4512991cd4.png)

**Comment by q1763868326** [comment]
Just pull the latest version of smart-doc to your local environment and run the install, and you should be all set.

**Comment by shalousun** [comment]
fix since 2.0.2
